port/src/agenttime_esp.c

```diff
@@ -8,7 +8,7 @@
 #include "freertos/event_groups.h"
 #include "esp_system.h"
 #include "esp_wifi.h"
-#include "esp_event_loop.h"
+#include "esp_event.h"
```

Comment:
This header doesn't appear to be used in this file, why not just remove it?

Reply:
There's other `#include`s in this file that aren't used.
Since I don't know the background of why they're there, I prefer to update rather than remove.
Unless someone from Espressif can confirm these can be removed.